docker-compose.yml

```diff
@@ -153,6 +153,7 @@ services:
       - "--proxy-fqdn=${APPS_FQDN}"
       - "--use-tls=${USE_TLS}"
       - "--admin-key=${GATEWAY_ADMIN_KEY}"
+      - "--posthog-key=${POSTHOG_KEY}"
```

Comment:
What should this look like when doing local development? We have some notes in DEVELOPING.md for Stripe and auth testing, and we might want to mention a few notes for Posthog too.

Reply:
Yeah, already reference it in `DEVELOPING.md` file.

Reply:
Since the eu hostname is already hard-coded, can we consider hard-coding the key as well, since it is public?